modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -1650,6 +1655,22 @@ private void _validateEnableObjectEntryHistory(
 		}
 	}
 
+	private void _validateExternalReferenceCode(
+			String externalReferenceCode, boolean modifiable,
+			String objectDefinitionName, boolean system)
```

Comment:
I think we could rename "objectDefinitionName" to "name" since we are using just "modifiable" and "system". What do you think?
Also, I searched for "objectDefinitionName" and I could not find so many examples. I think that we could call "name" since we are using this variable in the LocalService class.